### PR TITLE
Added Dutch (NL) translation

### PR DESCRIPTION
**Pull request by Roan-V**

### CODE REVIEW

File: src/main/resources/assets/purpurclient/lang/nl_nl.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "purpurclient.options.bee-count-in-debug": "Bijen Telling",
+  "purpurclient.options.bee-count-in-debug.tooltip": "Inschakelen voegt de hoeveelheid bijen toe aan het debugscherm (F3) bij het bekijken van bijenkorven en nesten",
+  "purpurclient.options.splash-screen": "Opstartscherm",
+  "purpurclient.options.splash-screen.tooltip": "Inschakelen zal het Purpur opstartscherm gebruiken in plaats van de originele Mojang versie",
+  "purpurclient.options.window-title": "Venstertitel",
+  "purpurclient.options.window-title.tooltip": "Inschakelen zal het Purpur venstertitel gebruiken in plaats van de originele Mojang titel",
+  "purpurclient.options.mobs": "Mobs",
+  "purpurclient.options.mobs.tooltip": "Opent het mob opties scherm",
+
+  "purpurclient.title.singleplayer": "%s in Singleplayer",
```
Comment (emielderckx) [inline src/main/resources/assets/purpurclient/lang/nl_nl.json:15]: Mojang translates singleplayer and multiplayer to `alleen spelen & samen spelen`

```suggestion
  "purpurclient.title.singleplayer": "%s in alleen spelen",
```

File: src/main/resources/assets/purpurclient/lang/nl_nl.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "purpurclient.title.multiplayer.realms": "Multiplayer (%s op Realms)",
```
Comment (emielderckx) [inline src/main/resources/assets/purpurclient/lang/nl_nl.json:16]: Mojang translates singleplayer and multiplayer to `alleen spelen & samen spelen`
```suggestion
  "purpurclient.title.multiplayer.realms": "Samen spelen (%s op Realms)",
```

File: src/main/resources/assets/purpurclient/lang/nl_nl.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "purpurclient.title.multiplayer.lan": "Multiplayer (%s in LAN)",
```
Comment (emielderckx) [inline src/main/resources/assets/purpurclient/lang/nl_nl.json:17]: Mojang translates singleplayer and multiplayer to `alleen spelen & samen spelen`
```suggestion
  "purpurclient.title.multiplayer.lan": "Samen spelen (%s op LAN)",
```

File: src/main/resources/assets/purpurclient/lang/nl_nl.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  "purpurclient.title.multiplayer.server": "Multiplayer (%s op %s)",
```
Comment (emielderckx) [inline src/main/resources/assets/purpurclient/lang/nl_nl.json:18]: Mojang translates singleplayer and multiplayer to `alleen spelen & samen spelen`
```suggestion
  "purpurclient.title.multiplayer.server": "Samen spelen (%s op %s)",
```

File: src/main/resources/assets/purpurclient/lang/nl_nl.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "purpurclient.title.multiplayer.unknown": "Multiplayer (%s op 3rd-party Server)",
```
Comment (emielderckx) [inline src/main/resources/assets/purpurclient/lang/nl_nl.json:19]: Mojang translates singleplayer and multiplayer to `alleen spelen & samen spelen`
```suggestion
  "purpurclient.title.multiplayer.unknown": "Samen spelen (%s op derden partij server)",
```

**Review by emielderckx — CHANGES_REQUESTED** [review]
Translating Dutch is awful, playing in Dutch should be forbidden 😞. 

(The translation is fine, just suggested some things Mojang does when you load the game in Dutch)

**Comment by Roan-V** [comment]
> Translating Dutch is awful, playing in Dutch should be forbidden 😞.
I wholeheartedly agree, this is of putting :P

**Review by emielderckx — APPROVED** [review]

**Pull request merged**